### ISSUE TITLE
Update Node version in Build / Test workflow

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA-TTS/tts.gsa.gov#103]
The current Pa11y and proposed Test workflows hard-code Node version 17.  Version 17 is no longer supported by either the Node or the Pages communities.

**Comment by wesley-dean-gsa** [comment]
While on BEARS, I used nvm (Node Version Manager) to manage the version of Node that's used:

https://github.com/GSA-TTS/usagov-benefits-eligibility/blob/main/.github/workflows/test.yml#L20-L27

Adopting this (or something like it) will allow us to use [nvm](https://github.com/nvm-sh/nvm) for Node version management; in the above case, the [.nvmrc](https://github.com/GSA-TTS/usagov-benefits-eligibility/blob/main/.nvmrc) only includes 18 so nvm will use the most recent version of Node version 18.

Currently, [Pages](https://cloud.gov/pages/documentation/node-on-pages/) uses nvm and supports the "active and maintenance LTS" releases of Node with the default version being 18 (specifically, 18.19.0).

**Comment by wesley-dean-gsa** [comment]
The Node release / support documentation is here: https://github.com/nodejs/release#release-schedule

**Comment by wesley-dean-gsa** [comment]
In #105, the Pa11y workflow was updated to use the live preview URLs generated by Cloud.gov Pages.  As a result, we're no longer building the site, so we don't need to install requirements from `package.json`.  All of that said, we're no longer tied to the version of Node required by the site, just the tool.  Pa11y currently supports versions 18 and 20.  As the current default for Cloud.gov Pages is Node 18.19.0, we just told it to use version 18 and let the system figure it out.

**tl;dr: the Pa11y workflow no longer needs to be updated.**

**Comment by wesley-dean-gsa** [comment]
FWIW, we went from 16.15.1 => 18.19.0.  This update required a change in the `.eleventy.js` configuration to make work locally; however, as a part of #136, we integrated that change which, apparently, is backwards-compatible.
